styleguide/source/_patterns/02-organisms/03-sections/11-state-data-info/state-data-info.twig

```diff
@@ -3,8 +3,7 @@
   {% include 'atoms-h3' %}
   {% set content = stateDataPage.stateDataSelect.states.alabama.paragraph %}
   {% include 'atoms-paragraph' %}
-  {% set src = stateDataPage.stateDataSelect.states.alabama.image %}
-  {% include 'atoms-landscape-3x2' %}
+  {% include 'molecules-figure' %}
```

Comment:
Don't want to create more work for you... but ideally this would have state-specific data from .json passed into the molecule, right?

Reply:
I realize that Downloads doesn't have data being passed to it either. I know this pattern is not being migrated to SG2 and also that you're the one implementing it. Up to you.

Reply:
@chelsiejohnston thank you for pointing this out, I really appreciate the thoughtful review and feedback. Since this is how they are both being used in the article template I'm going to say that it's acceptable in this case to use them as such based on the article template as the precedent.

@froboy @nstriedinger @Brian-Clement do you have thoughts?

Reply:
@labbydev I think that's a fine suggestion :)